### DISCUSSION TITLE
#16  Eye of the Tiger

### DISCUSSION THREAD

**Discussion by b-meet** [cssbattle/forum#112] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/82386196/134920345-58078e79-589d-4a62-b8f2-84a86880e64d.png)

AS you can see there is no visual difference between my solution and the challenge still the match percentage is 79%

**Comment by b-meet** [comment]
![image](https://user-images.githubusercontent.com/82386196/134921034-3f613818-71c0-46fd-9fe8-f7f4158d915b.png)

here is another preview, I am unable to figure out why is the percentage match is very little??😅

**Comment by SelenIT** [comment]
Are you testing this in a Chromium-based browser (Chrome, latest Edge, Opera etc.)? Aren't there any typos in the yellow color?

By the way, it's not necessary to type the `<body>...</body>` tags. The `body` element is created anyway regardless if these tags are present or not (it's a [standardized HTML feature](https://html.spec.whatwg.org/multipage/syntax.html#optional-tags)) and you can style it in any case.

**Reply by b-meet** [reply]
yes, it's chrome and there aren't any typo mistakes. body tag did not affect the results😐

**Comment by gunnarbittersmann** [comment]
> AS you can see there is no visual difference between my solution and the challenge still the match percentage is 79%

You might want to check your code in the [CSSBattle previewer](https://codepen.io/alexzaworski/full/vwmNKr). Do you see a difference there?

**Comment by rafehimself** [comment, marked as answer]
flex-box and grid were bugged, but they fixed it,
you should now get 100%

**Reply by b-meet** [reply]
well the percentage rose but not to 100% (93%). However, that may be fault by my side
